.pre-commit-config.yaml

```diff
@@ -33,7 +38,13 @@ repos:
     hooks:
       - id: "yamllint"
         args: ["--format", "parsable", "--strict"]
-        exclude: ".pre-commit-config.yaml"
+        # exclude: ".pre-commit-config.yaml"
+        exclude: |
+            (?x)^(
+                .pre-commit-config.yaml|
+                custom_components/ui_lovelace_minimalist/installation/.*\.yaml|
```

Comment:
Maybe this is a better way to ignore only certain sections explicitly, instead of ignoring complete files or folders:
https://yamllint.readthedocs.io/en/stable/disable_with_comments.html?highlight=jinja#disabling-checks-for-all-or-part-of-the-file